doc/roadmap.rst

```diff
@@ -124,12 +136,13 @@ bottom.
 
 #. Better interfaces for interactive development
 
-   * __repr__ and HTML visualisations of estimators :issue:`6323`
+   * |ss| __repr__ |se| and HTML visualisations of estimators
```

Comment:
the issue belongs to the repr not the html, right? And there's a PR for doing html by @thomasjpfan that should be mentioned here?